tests/unit/test-cmd/test.run.js

```diff
@@ -403,6 +434,7 @@ describe('run', () => {
       const args = {
         addonId: 'some-addon@test-suite',
         client,
+        // $FLOW_IGNORE: fake can return any kind of object and fake a defined set of methods for testing.
```

Comment:
I think that we can remove this flow suppress comment, by explicitly define the kill property on the `StubChildProcess`, e.g.:

```diff
diff --git a/tests/unit/test-cmd/test.run.js b/tests/unit/test-cmd/test.run.js
index f4dc88d..6898664 100644
--- a/tests/unit/test-cmd/test.run.js
+++ b/tests/unit/test-cmd/test.run.js
@@ -424,6 +424,7 @@ describe('run', () => {
     class StubChildProcess extends EventEmitter {
       stderr = new EventEmitter();
       stdout = new EventEmitter();
+      kill = sinon.spy(() => {});
     }

     function prepare() {
@@ -434,7 +435,6 @@ describe('run', () => {
       const args = {
         addonId: 'some-addon@test-suite',
         client,
-        // $FLOW_IGNORE: fake can return any kind of object and fake a defined set of methods for testing.
         firefoxProcess: new StubChildProcess(),
         profile: {},
         sourceDir: '/path/to/extension/source',
```